app/webpack/src/modules/proceedings.js

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 import { hide, show, pluralize } from '../helpers';
-const doneTypingInterval = 500; // time in ms, 500ms to make search work fairly quickly but avoid too many DB requests
+const doneTypingInterval = 10; // time in ms, 10ms to make search work fairly quickly but avoid too many DB requests
```

Comment:
I think if you're going to make the interval so small you may as well remove it completely - I really doubt anyone is going to be typing quicker than one letter per 0.01 seconds. I think this will mean that the search will run after every letter rather than after waiting for the user to finish typing - this makes the results refresh frequently giving the sort of flashing effect below the search box. The interval needs to be long enough to realistically cover the period between typing two letters, otherwise there is no point having it - happy to discuss